microsite/docs/RunWithDocker.md

```diff
@@ -41,3 +41,16 @@ If you would like to only run the API as a docker container, for example to depl
         cloudcarbonfootprint/api
 
 Then you can access the API at: http://localhost:4000/api/footprint?start=2021-01-01&end=2021-02-01
+
+### Run the Client as a Docker container
+
+If you would like to run the client as a docker container, you can pull and run it with these commands after running API as a docker container:
+
+    docker pull cloudcarbonfootprint/client:latest
+    
+    cd cloud-carbon-footprint && docker run \
```

Comment:
I don't think we need the `cd` command is necessary to run the container. Let's assume they're in the correct folder to avoid confusion, similar to how we did with the api instructions.
```suggestion
    docker run \
```

Reply:
Hi @4upz , I have made the required changes.

Reply:
Awesome, thanks @saranya2293 !